media-video/ffmpeg/ffmpeg-4.4.1-r2.ebuild

```diff
@@ -338,6 +339,11 @@ src_prepare() {
 	fi
 	default
 	echo 'include $(SRC_PATH)/ffbuild/libffmpeg.mak' >> Makefile || die
+
+	if usex vmaf; then
```

Comment:
Conditional patching should be avoided if possible as it's too hard to e.g miss when it stops applying.

also, use, not usex here if we were keeping it

Reply:
the patch is harmless so it can be applied without the if condition, will fix that !